packages/react-charts/src/components/Chart/Chart.tsx

```diff
@@ -236,11 +238,22 @@ export interface ChartProps extends VictoryChartProps {
   innerRadius?: number;
   /**
    * Allows legend items to wrap. A value of true allows the legend to wrap onto the next line
-   * if its container is not wide enough.
+   * if the chart is not wide enough.
+   *
+   * Note that the chart's SVG height and width are 100% by default, so it can be responsive itself. However, if you
+   * define the height and width of the chart's parent container, you must accommodate for extra legend height due to
+   * legend items wrapping onto the next line. When the height of the chart's parent container is too small, some legend
+   * items may not be visible.
    *
    * Note: This is overridden by the legendItemsPerRow property
    */
   legendAllowWrap?: boolean;
+  /**
+   * @beta If legendAllowWrap is true, this function will be called after the legend's itemsPerRow property has been
+   * calculated, based on available width. The value provided can be used to increase the chart's parent container
+   * height as legend items wrap onto the next line. If no adjustment is necessary, the value will be zero.
+   */
+  legendAllowWrapCallback?: (extraHeight: number) => void;
```

Comment:
What would you think about updating the type for `legendAllowWrap` to be something like `boolean  | (extraHeight: number) => void;` rather than adding a new prop explicitly for the CB that doesn't apply unless `legendAllowWrap` is true?

Reply:
@wise-king-sullyman I thought about `legendAllowWrap` accepting a boolean or a function, but thought that was confusing. There are some Victory properties that accept either a number or function, but those functions are used to fetch a value if you cannot provide one. I don't want to confuse this functionality with fetching a value.

Reply:
@wise-king-sullyman I merged legendAllowWrap and legendAllowWrapCallback.